src/core/manager.c

```diff
@@ -3879,6 +3881,20 @@ int manager_get_effective_environment(Manager *m, char ***ret) {
         return 0;
 }
 
+int manager_set_default_smack_process_label(Manager *m, const char *label) {
+        assert(m);
+
+#ifdef SMACK_DEFAULT_PROCESS_LABEL
+        if (!label)
+                return free_and_strdup(&m->default_smack_process_label, SMACK_DEFAULT_PROCESS_LABEL);
+        else
+#endif
+        if (streq(label, "/"))
```

Comment:
this will segfault if called with a NULL parameter, and SMACK_DEFAULT_PROCESS_LABEL is not defined.